docs/en/observability/application-logs.asciidoc

```diff
@@ -0,0 +1,219 @@
+[[application-logs]]
+= Application logs
+
+Application logs provide valuable insight into events that have occurred within your services and applications.
+Correlating these logs with application traces enables you to view the context in which a log event has occurred.
+
+Structured vs unstructured logs:
+
+Structured logging is where the application writes JSON objects directly--instead of
+parsing your logs into a structured object at ingest time.
+
+Why is structure so important in logs? So you can search and aggregate on individual fields within your logs.
+
+
+[float]
+[[ingest-application-logs]]
+== Ingest application logs
+
+There are four ways to ship your application logs to the {stack}.
+Your use case will determine the method that's right for you.
+
+[float]
+=== Plaintext logs with Filebeat
+
+**Requirements**
+
+* The Elastic APM agent for your programming language.
+* Raw, plain-text application logs
+* {filebeat}, to monitor the log files or locations that you specify, collects log events, filter and correlate multi-line logs, and forwards them to {es}
+
+**Pros**
+
+* All programming languages/frameworks are supported
+
+**Cons**
+
+* Must parse application logs to be useful—meaning writing and maintaining Grok patterns and spending CPU cycles on parsing
+
+Learn more: <<plaintext-with-filebeat>>
+
+[float]
+=== ECS loggers with Filebeat
+
+Elastic Common Schema (ECS) loggers format your logs into ECS-compatible JSON,
+which removes the need to manually parse logs.
+
+**Requirements**
+
+* The Elastic APM agent for your programming language
+* The Elastic ECS logger for your language or framework
+* {filebeat}, to monitor the log files or locations that you specify, collects log events, and forwards them to {es}
+
+**Pros**
+
+* Popular logging frameworks supported
+* Simplicity: no manual parsing
+* Decently human-readable JSON structure
+* APM Log correlation
+* Resilient in case of outages
+
+**Cons**
+
+* Not all frameworks are supported
+
+Learn more: <<ecs-with-filebeat>>
+
+[float]
+=== APM agent ECS reformatting with Filebeat
+
+Elastic APM agents can automatically reformat application logs to Elastic Common Schema (ECS) format
+without needing to add an ECS logger dependency.
+
+**Requirements**
+
+* The Elastic APM agent for your programming language
+* {filebeat}, to monitor the log files or locations that you specify, collects log events, and forwards them to {es}
+
+**Pros**
+
+**Cons**
+
+**Supported APM agents/languages**
+
+* Ruby
+* Java
+
+Learn more: <<ecs-reformatting-with-filebeat>>
+
+[float]
+=== APM agent direct logs
```

Comment:
```suggestion
=== APM agent logs sending
```

Reply:
Is this the official feature name?

Reply:
It's the name that we agreed on in agent specs : https://github.com/elastic/apm/blob/main/specs/agents/log-sending.md, so I would say that makes it official. Does it sounds OK to you as a native english speaker ?

Reply:
It's not my favorite, but I'm struggling to come up with an alternative, so I'll change it.

Reply:
My favorite one was "logs streaming" but sending seemed to be more general and not imply technical details. As usual, naming is always the biggest challenge, and even more fun when a non-native english speaker writes the spec :-).